2020-12-11 18:52:39,894 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 69938ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-11 21:51:26,546 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 10705684ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-12 00:51:34,422 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 10806860ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-12 03:51:41,412 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 10806810ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-12 04:51:52,621 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 3611114ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-12 06:52:53,042 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 7260403ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-12 09:21:50,739 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 8937682ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-12 10:21:59,872 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 3609125ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-12 15:52:01,987 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 19802108ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 03:39:21,856 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 42439038ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 04:01:28,152 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 844298ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 04:31:32,038 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 1803842ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 04:46:43,828 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 911784ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 05:16:48,524 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 1804692ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 05:46:52,506 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 1803978ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 08:44:03,043 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 10630533ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 08:58:56,687 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 853640ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 09:14:08,429 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 911728ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 09:44:06,883 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 1798442ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 10:14:10,014 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 1803129ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 10:44:16,027 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 1805997ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 10:46:08,873 ERROR 49771 nodejs.ClusterClientNoResponseError: client no response in 112841ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 49771
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 11:01:10,327 ERROR 26047 [-/127.0.0.1/-/5ms GET /queryContract?address=5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy] nodejs.TypeError: Cannot read property 'query' of undefined
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:44:61)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12

pid: 26047
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 11:01:35,039 ERROR 26047 [-/127.0.0.1/-/5ms GET /queryContract?address=5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy] nodejs.TypeError: Cannot read property 'query' of undefined
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:44:61)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12

pid: 26047
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 11:01:37,180 ERROR 26047 [-/127.0.0.1/-/1ms GET /queryContract?address=5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy] nodejs.TypeError: Cannot read property 'query' of undefined
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:44:61)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12

pid: 26047
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 11:02:25,442 ERROR 26047 [-/127.0.0.1/-/38098ms GET /queryContract?address=5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy] nodejs.TypeError: Cannot read property 'query' of undefined
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:44:61)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12

pid: 26047
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 11:04:21,101 ERROR 27845 [-/127.0.0.1/-/52530ms GET /queryContract?address=5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy] nodejs.TypeError: contract.query.get is not a function
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:44:67)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12

pid: 27845
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 11:07:07,322 ERROR 29742 [-/127.0.0.1/-/3087ms GET /queryContract?address=5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy] nodejs.Error: Expected 1 arguments to contract message 'balance_of', found 0
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)
    at decorateCall (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:69:10)
    at Object.send (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:101:73)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:195:184
    at Object.balanceOf (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:34:105)

pid: 29742
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 11:12:11,152 ERROR 31863 nodejs.ClusterClientNoResponseError: client no response in 71514ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 31863
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 11:12:35,124 ERROR 31976 [-/127.0.0.1/-/85259ms GET /queryContract?address=5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy] nodejs.ReferenceError: result is not defined
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:47:17)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:50
    at async xframe (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xframe.js:7:5)
    at async xssProtection (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xssProtection.js:7:5)
    at async nosniff (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/nosniff.js:8:5)
    at async noopen (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/noopen.js:8:5)
    at async session (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-session/index.js:41:7)
    at async bodyParser (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-bodyparser/index.js:95:5)
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/koa-static-cache/index.js:41:56

pid: 31976
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 11:13:38,655 ERROR 34431 [-/127.0.0.1/-/2057ms GET /queryContract?address=5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy] nodejs.ReferenceError: output is not defined
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:56:30)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:50
    at async xframe (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xframe.js:7:5)
    at async xssProtection (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xssProtection.js:7:5)
    at async nosniff (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/nosniff.js:8:5)
    at async noopen (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/noopen.js:8:5)
    at async session (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-session/index.js:41:7)
    at async bodyParser (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-bodyparser/index.js:95:5)
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/koa-static-cache/index.js:41:56

pid: 34431
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 11:18:05,854 ERROR 36345 nodejs.ClusterClientNoResponseError: client no response in 72483ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 36345
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:04:33,375 ERROR 66874 [-/127.0.0.1/-/2ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.ReferenceError: utils is not defined
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:73:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12

pid: 66874
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:05:55,927 ERROR 68229 [-/127.0.0.1/-/4ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.Error: specified phrase is not a valid mnemonic and is invalid as a raw seed at > 32 bytes
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/util/assert.js:34:11)
    at Keyring.createFromUri (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/keyring/keyring.js:238:26)
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:77:27)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18

pid: 68229
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:06:35,376 ERROR 68229 [-/127.0.0.1/-/3ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.ENOENTError: ENOENT: no such file or directory, scandir '../../erc20por.wasm'
    at Object.readdirSync (fs.js:854:3)
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:78:25)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
errno: -2
syscall: "scandir"
code: "ENOENT"
path: "../../erc20por.wasm"
name: "ENOENTError"
pid: 68229
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:08:12,444 ERROR 69580 [-/127.0.0.1/-/7539ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.ENOENTError: ENOENT: no such file or directory, scandir '../../erc20por.wasm'
    at Object.readdirSync (fs.js:854:3)
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:78:25)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
errno: -2
syscall: "scandir"
code: "ENOENT"
path: "../../erc20por.wasm"
name: "ENOENTError"
pid: 69580
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:10:29,255 ERROR 71037 [-/127.0.0.1/-/5805ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.ENOENTError: ENOENT: no such file or directory, open '../../erc20por.wasm'
    at Object.openSync (fs.js:440:3)
    at Object.readFileSync (fs.js:342:35)
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:78:25)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
errno: -2
syscall: "open"
code: "ENOENT"
path: "../../erc20por.wasm"
name: "ENOENTError"
pid: 71037
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:14:53,776 ERROR 73402 [-/127.0.0.1/-/2392ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.TypeError: Cannot read property 'startsWith' of undefined
    at Keyring.createFromUri (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/keyring/keyring.js:221:24)
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:78:27)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)

pid: 73402
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:15:10,340 ERROR 73402 [-/127.0.0.1/-/7530ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.TypeError: The value "r" is invalid for option "encoding"
    at assertEncoding (internal/fs/utils.js:75:11)
    at getOptions (internal/fs/utils.js:211:5)
    at Object.readFileSync (fs.js:340:13)
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:79:25)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
code: "ERR_INVALID_OPT_VALUE_ENCODING"
pid: 73402
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:17:05,748 ERROR 75300 [-/127.0.0.1/-/3643ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.ENOENTError: ENOENT: no such file or directory, open '../erc20por.wasm'
    at Object.openSync (fs.js:440:3)
    at Object.readFileSync (fs.js:342:35)
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:79:25)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
errno: -2
syscall: "open"
code: "ENOENT"
path: "../erc20por.wasm"
name: "ENOENTError"
pid: 75300
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:18:17,838 ERROR 76086 [-/127.0.0.1/-/3862ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.ENOENTError: ENOENT: no such file or directory, open '/erc20por.wasm'
    at Object.openSync (fs.js:440:3)
    at Object.readFileSync (fs.js:342:35)
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:79:25)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
errno: -2
syscall: "open"
code: "ENOENT"
path: "/erc20por.wasm"
name: "ENOENTError"
pid: 76086
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:19:16,414 ERROR 76778 [-/127.0.0.1/-/3882ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.ENOENTError: ENOENT: no such file or directory, open '../route.js'
    at Object.openSync (fs.js:440:3)
    at Object.readFileSync (fs.js:342:35)
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:79:25)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
errno: -2
syscall: "open"
code: "ENOENT"
path: "../route.js"
name: "ENOENTError"
pid: 76778
hostname: libaozhongdeMacBook-Pro.local

2020-12-13 12:22:04,108 ERROR 78478 [-/127.0.0.1/-/4144ms GET /publisContract?fromAddress=5HE7xcMe1g22ZthdFVQufw9FynEAvua7QfmW96phtkZBtJXU] nodejs.ENOENTError: ENOENT: no such file or directory, open '/Users/libaozhong/熊涛/node/attemper/app/erc20pro.wasm'
    at Object.openSync (fs.js:440:3)
    at Object.readFileSync (fs.js:342:35)
    at HomeController.publisContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:80:25)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
errno: -2
syscall: "open"
code: "ENOENT"
path: "/Users/libaozhong/熊涛/node/attemper/app/erc20pro.wasm"
name: "ENOENTError"
pid: 78478
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 15:37:44,650 ERROR 62354 [-/127.0.0.1/-/3ms POST /contractTransfor] nodejs.ReferenceError: fromAddress is not defined
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:73:34)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12

pid: 62354
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 15:38:40,224 ERROR 62942 [-/127.0.0.1/-/29ms POST /contractTransfor] nodejs.TypeError: Cannot read property 'startsWith' of undefined
    at Keyring.createFromUri (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/keyring/keyring.js:221:24)
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:75:27)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)

pid: 62942
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 15:39:02,883 ERROR 62942 [-/127.0.0.1/-/9203ms POST /contractTransfor] nodejs.TypeError: contract.tx.inc is not a function
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:85:8)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12

pid: 62942
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 15:40:07,317 ERROR 62941 nodejs.ClusterClientNoResponseError: client no response in 64425ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 62941
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 15:43:15,524 ERROR 65614 [-/127.0.0.1/-/7047ms POST /contractTransfor] nodejs.TypeError: contract.tx.transfor is not a function
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:85:8)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12

pid: 65614
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 15:44:11,428 ERROR 65614 [-/127.0.0.1/-/45494ms POST /contractTransfor] nodejs.TypeError: contract.tx.transfor is not a function
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:85:8)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12

pid: 65614
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 15:45:30,032 ERROR 66984 [-/127.0.0.1/-/5685ms POST /contractTransfor] nodejs.Error: Expected 2 arguments to contract message 'transfer', found 0
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at Contract.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:138:175)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:191:162
    at Object.transfer (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:38:67)
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:85:8)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)

pid: 66984
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 16:04:27,352 ERROR 79017 nodejs.ClusterClientNoResponseError: client no response in 70838ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 79017
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 16:07:52,812 ERROR 81785 [-/127.0.0.1/-/13851ms POST /contractTransfor] nodejs.Error: Expected 2 arguments to contract message 'transfer', found 0
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at Contract.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:138:175)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:191:162
    at Object.transfer (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:38:89)
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:123:34)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)

pid: 81785
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 16:17:11,963 ERROR 87072 [-/127.0.0.1/-/7895ms POST /contractTransfor] nodejs.Error: Expected 2 arguments to contract message 'transfer', found 0
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at Contract.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:138:175)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:191:162
    at Object.transfer (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:38:89)
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:123:34)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)

pid: 87072
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 16:24:36,710 ERROR 88842 nodejs.ClusterClientNoResponseError: client no response in 62989ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 88842
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 16:26:39,636 ERROR 93849 [-/127.0.0.1/-/6391ms POST /contractTransfor] nodejs.Error: createType(ContractCallRequest):: Struct: failed on origin: AccountId:: Unknown type passed to AccountId constructor
    at createTypeUnsafe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:71:11)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-core/index.js:373:70
    at Array.map (<anonymous>)
    at RpcCore._formatInputs (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-core/index.js:373:19)
    at callWithRegistry (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-core/index.js:257:27)
    at Observable._subscribe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-core/index.js:266:9)
    at Observable._trySubscribe (/Users/libaozhong/熊涛/node/attemper/node_modules/rxjs/internal/Observable.js:44:25)
    at Observable.subscribe (/Users/libaozhong/熊涛/node/attemper/node_modules/rxjs/internal/Observable.js:30:22)
    at Observable.ConnectableObservable.connect (/Users/libaozhong/熊涛/node/attemper/node_modules/rxjs/internal/observable/ConnectableObservable.js:47:18)
    at RefCountOperator.call (/Users/libaozhong/熊涛/node/attemper/node_modules/rxjs/internal/operators/refCount.js:33:49)

pid: 93849
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 16:42:00,825 ERROR 2697 nodejs.ClusterClientNoResponseError: client no response in 70571ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 2697
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 17:05:57,715 ERROR 19405 [-/127.0.0.1/-/13ms POST /contractTransfor] nodejs.TypeError: Cannot read property 'startsWith' of undefined
    at Keyring.createFromUri (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/keyring/keyring.js:221:24)
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:113:27)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)

pid: 19405
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 17:07:09,522 ERROR 20307 nodejs.unhandledRejectionError: 1010: Invalid Transaction: Transaction would exhaust the block limits
    at RpcCoder._checkError (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:84:13)
    at RpcCoder.decodeResponse (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:47:10)
    at WsProvider.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:212:90)
    at W3CWebSocket.value [as onmessage] (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:192:153)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/libaozhong/熊涛/node/attemper/node_modules/yaeti/lib/EventTarget.js:107:17)
    at W3CWebSocket.onMessage (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:234:14)
    at WebSocketConnection.<anonymous> (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:205:19)
    at WebSocketConnection.emit (events.js:210:5)
    at WebSocketConnection.processFrame (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:554:26)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:323:40
name: "unhandledRejectionError"
pid: 20307
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 17:07:40,424 ERROR 20307 nodejs.unhandledRejectionError: 1010: Invalid Transaction: Transaction would exhaust the block limits
    at RpcCoder._checkError (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:84:13)
    at RpcCoder.decodeResponse (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:47:10)
    at WsProvider.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:212:90)
    at W3CWebSocket.value [as onmessage] (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:192:153)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/libaozhong/熊涛/node/attemper/node_modules/yaeti/lib/EventTarget.js:107:17)
    at W3CWebSocket.onMessage (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:234:14)
    at WebSocketConnection.<anonymous> (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:205:19)
    at WebSocketConnection.emit (events.js:210:5)
    at WebSocketConnection.processFrame (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:554:26)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:323:40
name: "unhandledRejectionError"
pid: 20307
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 17:11:02,893 ERROR 20307 nodejs.unhandledRejectionError: 1010: Invalid Transaction: Transaction would exhaust the block limits
    at RpcCoder._checkError (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:84:13)
    at RpcCoder.decodeResponse (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:47:10)
    at WsProvider.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:212:90)
    at W3CWebSocket.value [as onmessage] (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:192:153)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/libaozhong/熊涛/node/attemper/node_modules/yaeti/lib/EventTarget.js:107:17)
    at W3CWebSocket.onMessage (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:234:14)
    at WebSocketConnection.<anonymous> (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:205:19)
    at WebSocketConnection.emit (events.js:210:5)
    at WebSocketConnection.processFrame (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:554:26)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:323:40
name: "unhandledRejectionError"
pid: 20307
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 17:14:15,971 ERROR 24877 [-/127.0.0.1/-/17ms POST /contractTransfor] nodejs.TypeError: Cannot read property 'startsWith' of undefined
    at Keyring.createFromUri (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/keyring/keyring.js:221:24)
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:113:27)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)

pid: 24877
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 17:14:28,007 ERROR 24877 [-/127.0.0.1/-/2ms POST /contractTransfor] nodejs.TypeError: Cannot read property 'startsWith' of undefined
    at Keyring.createFromUri (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/keyring/keyring.js:221:24)
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:113:27)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)

pid: 24877
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 17:14:33,107 ERROR 24877 nodejs.unhandledRejectionError: 1010: Invalid Transaction: Transaction would exhaust the block limits
    at RpcCoder._checkError (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:84:13)
    at RpcCoder.decodeResponse (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:47:10)
    at WsProvider.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:212:90)
    at W3CWebSocket.value [as onmessage] (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:192:153)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/libaozhong/熊涛/node/attemper/node_modules/yaeti/lib/EventTarget.js:107:17)
    at W3CWebSocket.onMessage (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:234:14)
    at WebSocketConnection.<anonymous> (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:205:19)
    at WebSocketConnection.emit (events.js:210:5)
    at WebSocketConnection.processFrame (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:554:26)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:323:40
name: "unhandledRejectionError"
pid: 24877
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 17:40:41,361 ERROR 24877 nodejs.unhandledRejectionError: 1010: Invalid Transaction: Transaction would exhaust the block limits
    at RpcCoder._checkError (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:84:13)
    at RpcCoder.decodeResponse (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:47:10)
    at WsProvider.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:212:90)
    at W3CWebSocket.value [as onmessage] (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:192:153)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/libaozhong/熊涛/node/attemper/node_modules/yaeti/lib/EventTarget.js:107:17)
    at W3CWebSocket.onMessage (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:234:14)
    at WebSocketConnection.<anonymous> (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:205:19)
    at WebSocketConnection.emit (events.js:210:5)
    at WebSocketConnection.processFrame (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:554:26)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:323:40
name: "unhandledRejectionError"
pid: 24877
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 17:56:44,190 ERROR 24877 nodejs.unhandledRejectionError: 1010: Invalid Transaction: Transaction would exhaust the block limits
    at RpcCoder._checkError (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:84:13)
    at RpcCoder.decodeResponse (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:47:10)
    at WsProvider.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:212:90)
    at W3CWebSocket.value [as onmessage] (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:192:153)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/libaozhong/熊涛/node/attemper/node_modules/yaeti/lib/EventTarget.js:107:17)
    at W3CWebSocket.onMessage (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:234:14)
    at WebSocketConnection.<anonymous> (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:205:19)
    at WebSocketConnection.emit (events.js:210:5)
    at WebSocketConnection.processFrame (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:554:26)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:323:40
name: "unhandledRejectionError"
pid: 24877
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 18:09:47,143 ERROR 60186 nodejs.unhandledRejectionError: 1010: Invalid Transaction: Transaction would exhaust the block limits
    at RpcCoder._checkError (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:84:13)
    at RpcCoder.decodeResponse (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:47:10)
    at WsProvider.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:212:90)
    at W3CWebSocket.value [as onmessage] (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:192:153)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/libaozhong/熊涛/node/attemper/node_modules/yaeti/lib/EventTarget.js:107:17)
    at W3CWebSocket.onMessage (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:234:14)
    at WebSocketConnection.<anonymous> (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:205:19)
    at WebSocketConnection.emit (events.js:210:5)
    at WebSocketConnection.processFrame (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:554:26)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:323:40
name: "unhandledRejectionError"
pid: 60186
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 18:18:51,830 ERROR 66217 [-/127.0.0.1/-/27ms POST /contractTransfor] nodejs.TypeError: Cannot read property 'startsWith' of undefined
    at Keyring.createFromUri (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/keyring/keyring.js:221:24)
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:113:27)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)

pid: 66217
hostname: libaozhongdeMacBook-Pro.local

2020-12-14 18:18:57,372 ERROR 66217 [-/127.0.0.1/-/2ms POST /contractTransfor] nodejs.TypeError: Cannot read property 'startsWith' of undefined
    at Keyring.createFromUri (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/keyring/keyring.js:221:24)
    at HomeController.contractTransfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:113:27)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)

pid: 66217
hostname: libaozhongdeMacBook-Pro.local

2020-12-15 16:37:02,997 ERROR 65931 [-/127.0.0.1/-/0ms GET /] nodejs.ECONNREFUSEDError: connect ECONNREFUSED 127.0.0.1:3306
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1128:14)
    --------------------
    at Protocol._enqueue (/Users/libaozhong/熊涛/node/attemper/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/Users/libaozhong/熊涛/node/attemper/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at PoolConnection.connect (/Users/libaozhong/熊涛/node/attemper/node_modules/mysql/lib/Connection.js:116:18)
    at Pool.getConnection (/Users/libaozhong/熊涛/node/attemper/node_modules/mysql/lib/Pool.js:48:16)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/ali-rds/node_modules/pify/index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (/Users/libaozhong/熊涛/node/attemper/node_modules/ali-rds/node_modules/pify/index.js:12:10)
    at Pool.ret [as getConnection] (/Users/libaozhong/熊涛/node/attemper/node_modules/ali-rds/node_modules/pify/index.js:56:34)
    at Pool.query (/Users/libaozhong/熊涛/node/attemper/node_modules/mysql/lib/Pool.js:202:8)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/ali-rds/node_modules/pify/index.js:29:7
    sql: select now() as currentTime;
errno: "ECONNREFUSED"
code: "ECONNREFUSED"
syscall: "connect"
address: "127.0.0.1"
port: 3306
fatal: true
name: "ECONNREFUSEDError"
pid: 65931
hostname: libaozhongdeMacBook-Pro.local

2020-12-15 16:57:49,656 ERROR 72188 [-/127.0.0.1/-/69ms GET /queryContract?address=5G1QfcZWMREnH52Nngsh3GwHBeJCs9faPudC4s9NHL9qeu7Q] nodejs.Error: Invalid character
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:6:21)
    at parseBase (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:269:7)
    at BN._parseBase (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:293:14)
    at BN.init [as _init] (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:99:12)
    at new BN (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:39:12)
    at bnToBn (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/util/bn/toBn.js:48:87)
    at _formatBalance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/util/format/formatBalance.js:28:32)
    at Object.util.formatBalance (/Users/libaozhong/熊涛/node/attemper/app/extend/helper.js:18:18)
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:102:20)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

pid: 72188
hostname: libaozhongdeMacBook-Pro.local

2020-12-15 16:57:51,490 ERROR 72188 [-/127.0.0.1/-/31ms GET /queryContract?address=5G1QfcZWMREnH52Nngsh3GwHBeJCs9faPudC4s9NHL9qeu7Q] nodejs.Error: Invalid character
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:6:21)
    at parseBase (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:269:7)
    at BN._parseBase (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:293:14)
    at BN.init [as _init] (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:99:12)
    at new BN (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:39:12)
    at bnToBn (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/util/bn/toBn.js:48:87)
    at _formatBalance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/util/format/formatBalance.js:28:32)
    at Object.util.formatBalance (/Users/libaozhong/熊涛/node/attemper/app/extend/helper.js:18:18)
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:102:20)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

pid: 72188
hostname: libaozhongdeMacBook-Pro.local

2020-12-15 16:59:37,920 ERROR 80284 [-/127.0.0.1/-/57ms GET /queryContract?address=5G1QfcZWMREnH52Nngsh3GwHBeJCs9faPudC4s9NHL9qeu7Q] nodejs.Error: Assertion failed
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:6:21)
    at BN._initArray (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:145:5)
    at BN.init [as _init] (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:82:19)
    at new BN (/Users/libaozhong/熊涛/node/attemper/node_modules/bn.js/lib/bn.js:39:12)
    at bnToBn (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/util/bn/toBn.js:48:87)
    at _formatBalance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/util/format/formatBalance.js:28:32)
    at Object.util.formatBalance (/Users/libaozhong/熊涛/node/attemper/app/extend/helper.js:18:18)
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:102:20)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:50

pid: 80284
hostname: libaozhongdeMacBook-Pro.local

2020-12-16 17:04:56,357 ERROR 86176 nodejs.unhandledRejectionError: [object Object]
    at Application._unhandledRejectionHandler (/Users/libaozhong/熊涛/node/attemper/node_modules/egg/lib/egg.js:344:24)
    at process.emit (events.js:210:5)
    at processPromiseRejections (internal/process/promises.js:201:33)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
name: "unhandledRejectionError"
pid: 86176
hostname: libaozhongdeMacBook-Pro.local

2020-12-16 17:08:52,845 ERROR 88047 [-/127.0.0.1/-/53ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNnowYachvmtNfnN7hDuVcc8] nodejs.TypeError: Cannot read property 'toString' of null
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:113:41)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/koa-static-cache/index.js:53:36
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:50
    at async xframe (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xframe.js:7:5)
    at async xssProtection (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xssProtection.js:7:5)
    at async nosniff (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/nosniff.js:8:5)
    at async noopen (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/noopen.js:8:5)
    at async session (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-session/index.js:41:7)
    at async bodyParser (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-bodyparser/index.js:95:5)

pid: 88047
hostname: libaozhongdeMacBook-Pro.local

2020-12-16 17:09:44,190 ERROR 88047 [-/127.0.0.1/-/26127ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNnowYachvmtNfnN7hDuVcc8] nodejs.TypeError: Cannot read property 'toString' of null
    at HomeController.queryContract (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:113:41)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/koa-static-cache/index.js:53:36
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:50
    at async xframe (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xframe.js:7:5)
    at async xssProtection (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xssProtection.js:7:5)
    at async nosniff (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/nosniff.js:8:5)
    at async noopen (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/noopen.js:8:5)
    at async session (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-session/index.js:41:7)
    at async bodyParser (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-bodyparser/index.js:95:5)

pid: 88047
hostname: libaozhongdeMacBook-Pro.local

2020-12-17 14:47:15,545 ERROR 85821 nodejs.unhandledRejectionError: [object Object]
    at Application._unhandledRejectionHandler (/Users/libaozhong/熊涛/node/attemper/node_modules/egg/lib/egg.js:344:24)
    at process.emit (events.js:210:5)
    at processPromiseRejections (internal/process/promises.js:201:33)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
name: "unhandledRejectionError"
pid: 85821
hostname: libaozhongdeMacBook-Pro.local

2020-12-17 14:57:12,980 ERROR 85821 application still doesn't ready after 600000 ms.
2020-12-17 15:05:43,943 ERROR 97639 nodejs.unhandledRejectionError: [object Object]
    at Application._unhandledRejectionHandler (/Users/libaozhong/熊涛/node/attemper/node_modules/egg/lib/egg.js:344:24)
    at process.emit (events.js:210:5)
    at processPromiseRejections (internal/process/promises.js:201:33)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
name: "unhandledRejectionError"
pid: 97639
hostname: libaozhongdeMacBook-Pro.local

2020-12-17 15:12:18,925 ERROR 2301 nodejs.unhandledRejectionError: [object Object]
    at Application._unhandledRejectionHandler (/Users/libaozhong/熊涛/node/attemper/node_modules/egg/lib/egg.js:344:24)
    at process.emit (events.js:210:5)
    at processPromiseRejections (internal/process/promises.js:201:33)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
name: "unhandledRejectionError"
pid: 2301
hostname: libaozhongdeMacBook-Pro.local

2020-12-17 15:16:02,860 ERROR 4656 nodejs.unhandledRejectionError: [object Object]
    at Application._unhandledRejectionHandler (/Users/libaozhong/熊涛/node/attemper/node_modules/egg/lib/egg.js:344:24)
    at process.emit (events.js:210:5)
    at processPromiseRejections (internal/process/promises.js:201:33)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
name: "unhandledRejectionError"
pid: 4656
hostname: libaozhongdeMacBook-Pro.local

2020-12-17 16:06:05,264 ERROR 36377 nodejs.unhandledRejectionError: [object Object]
    at Application._unhandledRejectionHandler (/Users/libaozhong/熊涛/node/attemper/node_modules/egg/lib/egg.js:344:24)
    at process.emit (events.js:210:5)
    at processPromiseRejections (internal/process/promises.js:201:33)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
name: "unhandledRejectionError"
pid: 36377
hostname: libaozhongdeMacBook-Pro.local

2020-12-17 18:56:09,522 ERROR 46098 nodejs.unhandledRejectionError: [object Object]
    at Application._unhandledRejectionHandler (/Users/libaozhong/熊涛/node/attemper/node_modules/egg/lib/egg.js:344:24)
    at process.emit (events.js:210:5)
    at processPromiseRejections (internal/process/promises.js:201:33)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
name: "unhandledRejectionError"
pid: 46098
hostname: libaozhongdeMacBook-Pro.local

2020-12-17 18:56:48,196 ERROR 46540 nodejs.unhandledRejectionError: [object Object]
    at Application._unhandledRejectionHandler (/Users/libaozhong/熊涛/node/attemper/node_modules/egg/lib/egg.js:344:24)
    at process.emit (events.js:210:5)
    at processPromiseRejections (internal/process/promises.js:201:33)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
name: "unhandledRejectionError"
pid: 46540
hostname: libaozhongdeMacBook-Pro.local

2020-12-17 20:06:20,318 ERROR 91384 nodejs.unhandledRejectionError: [object Object]
    at Application._unhandledRejectionHandler (/Users/libaozhong/熊涛/node/attemper/node_modules/egg/lib/egg.js:344:24)
    at process.emit (events.js:210:5)
    at processPromiseRejections (internal/process/promises.js:201:33)
    at processTicksAndRejections (internal/process/task_queues.js:94:32)
name: "unhandledRejectionError"
pid: 91384
hostname: libaozhongdeMacBook-Pro.local

2020-12-18 13:56:36,536 ERROR 72458 [-/127.0.0.1/-/7ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8] nodejs.Error: createType(AccountId):: Decoding 5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8: Invalid decoded address checksum
    at createTypeUnsafe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:71:11)
    at createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:83:10)
    at MetaRegistry.createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/registry.js:257:39)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:149:36
    at Array.map (<anonymous>)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:147:137)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)

pid: 72458
hostname: libaozhongdeMacBook-Pro.local

2020-12-18 13:57:53,582 ERROR 72458 [-/127.0.0.1/-/11ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8] nodejs.Error: createType(AccountId):: Decoding 5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8: Invalid decoded address checksum
    at createTypeUnsafe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:71:11)
    at createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:83:10)
    at MetaRegistry.createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/registry.js:257:39)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:149:36
    at Array.map (<anonymous>)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:147:137)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)

pid: 72458
hostname: libaozhongdeMacBook-Pro.local

2020-12-18 13:57:53,591 ERROR 72458 [-/127.0.0.1/-/4ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8] nodejs.Error: createType(AccountId):: Decoding 5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8: Invalid decoded address checksum
    at createTypeUnsafe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:71:11)
    at createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:83:10)
    at MetaRegistry.createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/registry.js:257:39)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:149:36
    at Array.map (<anonymous>)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:147:137)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)

pid: 72458
hostname: libaozhongdeMacBook-Pro.local

2020-12-18 13:57:53,596 ERROR 72458 [-/127.0.0.1/-/3ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8] nodejs.Error: createType(AccountId):: Decoding 5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8: Invalid decoded address checksum
    at createTypeUnsafe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:71:11)
    at createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:83:10)
    at MetaRegistry.createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/registry.js:257:39)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:149:36
    at Array.map (<anonymous>)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:147:137)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)

pid: 72458
hostname: libaozhongdeMacBook-Pro.local

2020-12-18 13:57:53,601 ERROR 72458 [-/127.0.0.1/-/4ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8] nodejs.Error: createType(AccountId):: Decoding 5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8: Invalid decoded address checksum
    at createTypeUnsafe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:71:11)
    at createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:83:10)
    at MetaRegistry.createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/registry.js:257:39)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:149:36
    at Array.map (<anonymous>)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:147:137)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)

pid: 72458
hostname: libaozhongdeMacBook-Pro.local

2020-12-18 13:57:57,790 ERROR 72458 [-/127.0.0.1/-/4ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8] nodejs.Error: createType(AccountId):: Decoding 5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8: Invalid decoded address checksum
    at createTypeUnsafe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:71:11)
    at createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:83:10)
    at MetaRegistry.createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/registry.js:257:39)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:149:36
    at Array.map (<anonymous>)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:147:137)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)

pid: 72458
hostname: libaozhongdeMacBook-Pro.local

2020-12-18 13:57:58,677 ERROR 72458 [-/127.0.0.1/-/4ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8] nodejs.Error: createType(AccountId):: Decoding 5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8: Invalid decoded address checksum
    at createTypeUnsafe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:71:11)
    at createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:83:10)
    at MetaRegistry.createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/registry.js:257:39)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:149:36
    at Array.map (<anonymous>)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:147:137)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)

pid: 72458
hostname: libaozhongdeMacBook-Pro.local

2020-12-18 13:58:08,508 ERROR 72458 [-/127.0.0.1/-/4ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8] nodejs.Error: createType(AccountId):: Decoding 5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8: Invalid decoded address checksum
    at createTypeUnsafe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:71:11)
    at createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:83:10)
    at MetaRegistry.createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/registry.js:257:39)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:149:36
    at Array.map (<anonymous>)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:147:137)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)

pid: 72458
hostname: libaozhongdeMacBook-Pro.local

2020-12-18 13:58:48,485 ERROR 74455 [-/127.0.0.1/-/5452ms GET /queryContract?address=5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8] nodejs.Error: createType(AccountId):: Decoding 5FvX6EDv7ndjSrj5b4hDUES9uNfowYachvmtNfnN7hDuVcc8: Invalid decoded address checksum
    at createTypeUnsafe (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:71:11)
    at createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/createType.js:83:10)
    at MetaRegistry.createType (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/types/create/registry.js:257:39)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:149:36
    at Array.map (<anonymous>)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:147:137)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)

pid: 74455
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:10:38,731 ERROR 92069 [-/127.0.0.1/-/28ms POST /allowance] nodejs.Error: specified phrase is not a valid mnemonic and is invalid as a raw seed at > 32 bytes
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/util/assert.js:34:11)
    at Keyring.createFromUri (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/node_modules/@polkadot/keyring/keyring.js:238:26)
    at HomeController.allowanceRequest (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:172:27)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 92069
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:14:56,285 ERROR 92068 nodejs.ClusterClientNoResponseError: client no response in 78786ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 92068
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:15:57,768 ERROR 97064 [-/127.0.0.1/-/18ms POST /allowance] nodejs.Error: Expected 2 arguments to contract message 'allowance', found 0
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)
    at decorateCall (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:69:10)
    at Object.send (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:101:73)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:195:184
    at Object.allowance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:34:105)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 97064
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:16:31,398 ERROR 97409 [-/127.0.0.1/-/19ms POST /allowance] nodejs.Error: Expected 2 arguments to contract message 'allowance', found 1
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)
    at decorateCall (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:69:10)
    at Object.send (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:101:73)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:195:184
    at Object.allowance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:34:105)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 97409
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:16:32,211 ERROR 97409 [-/127.0.0.1/-/2ms POST /allowance] nodejs.Error: Expected 2 arguments to contract message 'allowance', found 1
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)
    at decorateCall (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:69:10)
    at Object.send (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:101:73)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:195:184
    at Object.allowance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:34:105)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 97409
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:16:32,996 ERROR 97409 [-/127.0.0.1/-/1ms POST /allowance] nodejs.Error: Expected 2 arguments to contract message 'allowance', found 1
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)
    at decorateCall (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:69:10)
    at Object.send (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:101:73)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:195:184
    at Object.allowance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:34:105)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 97409
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:16:33,684 ERROR 97409 [-/127.0.0.1/-/2ms POST /allowance] nodejs.Error: Expected 2 arguments to contract message 'allowance', found 1
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)
    at decorateCall (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:69:10)
    at Object.send (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:101:73)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:195:184
    at Object.allowance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:34:105)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 97409
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:16:34,581 ERROR 97409 [-/127.0.0.1/-/3ms POST /allowance] nodejs.Error: Expected 2 arguments to contract message 'allowance', found 1
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)
    at decorateCall (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:69:10)
    at Object.send (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:101:73)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:195:184
    at Object.allowance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:34:105)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 97409
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:16:35,443 ERROR 97409 [-/127.0.0.1/-/2ms POST /allowance] nodejs.Error: Expected 2 arguments to contract message 'allowance', found 1
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)
    at decorateCall (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:69:10)
    at Object.send (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:101:73)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:195:184
    at Object.allowance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:34:105)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 97409
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:16:36,164 ERROR 97409 [-/127.0.0.1/-/1ms POST /allowance] nodejs.Error: Expected 2 arguments to contract message 'allowance', found 1
    at assert (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/node_modules/@polkadot/util/assert.js:34:11)
    at Abi.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:146:26)
    at Object.toU8a (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/Abi.js:109:115)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:165:32
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:73:26
    at new Promise (<anonymous>)
    at decorateCall (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:69:10)
    at Object.send (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api/promise/index.js:101:73)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:195:184
    at Object.allowance (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/api-contract/base/Contract.js:34:105)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 97409
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:18:42,934 ERROR 98857 [-/127.0.0.1/-/147ms POST /allowance] nodejs.TypeError: tx.toHuman is not a function
    at HomeController.allowanceRequest (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:181:18)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/koa-static-cache/index.js:39:63
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:50
    at async xframe (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xframe.js:7:5)
    at async xssProtection (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xssProtection.js:7:5)
    at async nosniff (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/nosniff.js:8:5)
    at async noopen (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/noopen.js:8:5)
    at async session (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-session/index.js:41:7)
    at async bodyParser (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-bodyparser/index.js:95:5)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 98857
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:18:46,189 ERROR 98857 [-/127.0.0.1/-/118ms POST /allowance] nodejs.TypeError: tx.toHuman is not a function
    at HomeController.allowanceRequest (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:181:18)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/koa-static-cache/index.js:39:63
    at async /Users/libaozhong/熊涛/node/attemper/node_modules/egg-development/app/middleware/egg_loader_trace.js:9:50
    at async xframe (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xframe.js:7:5)
    at async xssProtection (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/xssProtection.js:7:5)
    at async nosniff (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/nosniff.js:8:5)
    at async noopen (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-security/lib/middlewares/noopen.js:8:5)
    at async session (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-session/index.js:41:7)
    at async bodyParser (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-bodyparser/index.js:95:5)
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 98857
hostname: libaozhongdeMacBook-Pro.local

2020-12-19 17:20:42,364 ERROR 99312 nodejs.ClusterClientNoResponseError: client no response in 67040ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 99312
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 18:56:13,613 ERROR 49191 [-/127.0.0.1/-/38ms POST /transfor] nodejs.TypeError: Cannot destructure property `fromAddress` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:328:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 49191
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 18:56:33,536 ERROR 49191 [-/127.0.0.1/-/1ms POST /transfor] nodejs.TypeError: Cannot destructure property `fromAddress` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:328:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 49191
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 18:57:13,756 ERROR 50411 [-/127.0.0.1/-/14ms POST /transfor] nodejs.TypeError: Cannot destructure property `balances` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:328:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 50411
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 18:57:45,522 ERROR 50411 [-/127.0.0.1/-/6ms POST /transfor] nodejs.TypeError: Cannot destructure property `balances` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:328:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 50411
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 18:57:48,132 ERROR 50411 [-/127.0.0.1/-/1ms POST /transfor] nodejs.TypeError: Cannot destructure property `balances` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:328:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 50411
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 18:57:48,864 ERROR 50411 [-/127.0.0.1/-/1ms POST /transfor] nodejs.TypeError: Cannot destructure property `balances` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:328:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 50411
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 18:57:49,702 ERROR 50411 [-/127.0.0.1/-/1ms POST /transfor] nodejs.TypeError: Cannot destructure property `balances` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:328:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 50411
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 18:58:35,503 ERROR 51303 [-/127.0.0.1/-/15ms POST /transfor] nodejs.TypeError: Cannot destructure property `balances` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:328:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 51303
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 18:59:14,241 ERROR 51303 [-/127.0.0.1/-/5ms POST /transfor] nodejs.TypeError: Cannot destructure property `balances` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:328:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 51303
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 19:00:34,591 ERROR 52532 [-/127.0.0.1/-/32ms POST /transfor] nodejs.TypeError: Cannot destructure property `balances` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:329:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 52532
hostname: libaozhongdeMacBook-Pro.local

2020-12-27 19:00:35,601 ERROR 52532 [-/127.0.0.1/-/2ms POST /transfor] nodejs.TypeError: Cannot destructure property `balances` of 'undefined' or 'null'.
    at HomeController.transfor (/Users/libaozhong/熊涛/node/attemper/app/controller/home.js:329:11)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at Object.classControllerMiddleware (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/loader/mixin/controller.js:87:20)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/utils.js:12:21)
    at wrappedController (/Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/egg_router.js:322:18)
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at next (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:45:18)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/@eggjs/router/lib/router.js:190:18
    at dispatch (/Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:44:32)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/koa-compose/index.js:36:12
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 52532
hostname: libaozhongdeMacBook-Pro.local

2020-12-31 11:11:44,545 ERROR 6106 nodejs.ClusterClientNoResponseError: client no response in 69528ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 6106
hostname: libaozhongdeMacBook-Pro.local

2020-12-31 11:13:56,980 ERROR 8595 [-/127.0.0.1/-/0ms GET /] nodejs.Error: [egg-swagger-doc] error at get:/queryContract ,the type of request parameter does not exit
    at generateParameters (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:437:13)
    at generateRequest (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:331:23)
    at getTag_Path (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:137:36)
    at buildDocument (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:50:14)
    at documentInit (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:497:7)
    at module.exports.swaggerInit (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/index.js:10:3)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/app.js:8:5
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:42)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/lifecycle.js:256:13
    at processTicksAndRejections (internal/process/task_queues.js:75:11)

pid: 8595
hostname: libaozhongdeMacBook-Pro.local

2020-12-31 11:14:41,425 ERROR 9052 [-/127.0.0.1/-/0ms GET /] nodejs.Error: [egg-swagger-doc] error at get:/queryContract ,the type of request parameter does not exit
    at generateParameters (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:437:13)
    at generateRequest (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:331:23)
    at getTag_Path (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:137:36)
    at buildDocument (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:50:14)
    at documentInit (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:497:7)
    at module.exports.swaggerInit (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/index.js:10:3)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/app.js:8:5
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:42)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/lifecycle.js:256:13
    at processTicksAndRejections (internal/process/task_queues.js:75:11)

pid: 9052
hostname: libaozhongdeMacBook-Pro.local

2020-12-31 11:20:18,617 ERROR 12643 [-/127.0.0.1/-/0ms GET /] nodejs.Error: [egg-swagger-doc] app/contract/database.js: AccountList.list.itemType is necessary
    at buildDefinition (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/contract/index.js:97:17)
    at Object.getDefinitions (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/contract/index.js:183:23)
    at buildDocument (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:45:30)
    at documentInit (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/document/index.js:497:7)
    at module.exports.swaggerInit (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/lib/index.js:10:3)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/egg-swagger-doc/app.js:8:5
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:42)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/lifecycle.js:256:13
    at processTicksAndRejections (internal/process/task_queues.js:75:11)

pid: 12643
hostname: libaozhongdeMacBook-Pro.local

2020-12-31 11:23:21,405 ERROR 11075 nodejs.ClusterClientNoResponseError: client no response in 75334ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/libaozhong/熊涛/node/attemper/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
name: "ClusterClientNoResponseError"
pid: 11075
hostname: libaozhongdeMacBook-Pro.local

2020-12-31 11:27:26,495 ERROR 17243 [-/127.0.0.1/-/0ms GET /] nodejs.TypeError: Cannot read property 'derive' of undefined
    at AppBootHook.serverDidReady (/Users/libaozhong/熊涛/node/attemper/app.js:56:24)
    at Object.callFn (/Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/utils/index.js:44:21)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/egg-core/lib/lifecycle.js:207:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

pid: 17243
hostname: libaozhongdeMacBook-Pro.local

2021-01-01 11:47:44,542 ERROR 94667 [-/127.0.0.1/-/207ms POST /allowance] nodejs.Error: 1014: Priority is too low: (300750001392 vs 300750001392): The transaction has too low priority to replace another transaction already in the pool.
    at RpcCoder._checkError (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:84:13)
    at RpcCoder.decodeResponse (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:47:10)
    at WsProvider.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:212:90)
    at W3CWebSocket.value [as onmessage] (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:192:153)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/libaozhong/熊涛/node/attemper/node_modules/yaeti/lib/EventTarget.js:107:17)
    at W3CWebSocket.onMessage (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:234:14)
    at WebSocketConnection.<anonymous> (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:205:19)
    at WebSocketConnection.emit (events.js:210:5)
    at WebSocketConnection.processFrame (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:554:26)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:323:40
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 94667
hostname: libaozhongdeMacBook-Pro.local

2021-01-01 11:47:45,466 ERROR 94667 [-/127.0.0.1/-/224ms POST /allowance] nodejs.Error: 1014: Priority is too low: (300750001392 vs 300750001392): The transaction has too low priority to replace another transaction already in the pool.
    at RpcCoder._checkError (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:84:13)
    at RpcCoder.decodeResponse (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/coder/index.js:47:10)
    at WsProvider.value (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:212:90)
    at W3CWebSocket.value [as onmessage] (/Users/libaozhong/熊涛/node/attemper/node_modules/@polkadot/rpc-provider/ws/index.js:192:153)
    at W3CWebSocket._dispatchEvent [as dispatchEvent] (/Users/libaozhong/熊涛/node/attemper/node_modules/yaeti/lib/EventTarget.js:107:17)
    at W3CWebSocket.onMessage (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:234:14)
    at WebSocketConnection.<anonymous> (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/W3CWebSocket.js:205:19)
    at WebSocketConnection.emit (events.js:210:5)
    at WebSocketConnection.processFrame (/Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:554:26)
    at /Users/libaozhong/熊涛/node/attemper/node_modules/websocket/lib/WebSocketConnection.js:323:40
headers: {"Access-Control-Allow-Origin":"*","Access-Control-Allow-Credentials":"true","vary":"Origin"}
pid: 94667
hostname: libaozhongdeMacBook-Pro.local

